src/riscv.c

```diff
@@ -77,6 +100,8 @@ riscv_word_t rv_get_reg(riscv_t *rv, uint32_t reg)
     return ~0U;
 }
 
+#define BLOCK_MAP_CAPACITY_BITS 10
```

Comment:
Avoid duplicating `BLOCK_MAP_CAPACITY_BITS` and `BLOCK_CAPACITY_BITS` definitions.